### PR TITLE
Fix perl warnings

### PR DESCRIPTION
**Pull request by bldewolf**
As per #289

### CODE REVIEW

File: macros/parserPopUp.pl [hunk 1]
```diff
@@ -168,7 +168,7 @@ sub MENU {
     $menu = qq!<select class="pg-select" name="$name" id="$name" aria-label="$label" size="1">\n!;
     foreach my $item (@list) {
       my $selected = ($item eq $answer_value) ? " selected" : "";
-      my $option = $self->quoteHTML($item,true);
+      my $option = $self->quoteHTML($item,"true");
```
Comment (dpvc) [inline macros/parserPopUp.pl]: Here, `"true"` should be changed to `1`.

**Comment by jwj61** [comment]
This has a conflict with niceTables.pl

**Comment by bldewolf** [comment]
Resolved conflict and made changes requested by @dpvc

**Comment by pstaabp** [comment]
@bldewolf, @mgage, @dpvc    I just read through the thread for #289, do any of you a list of problems that generate the warnings, so I can test if these fix the warnings?

**Comment by mgage** [comment]
Hi Peter,

Here is part of the needed reference below.

I don’t have specific problems. I thought there was a pull request of fixed problems to the webwork-open-problem-library but I can’t find it at the moment.  I believe that the warnings only show up
in the server error_log — they aren’t printed to the HTML output.

Take care,

Mike

> On Jun 21, 2017, at 4:38 PM, Peter Staab <notifications@github.com> wrote:
> 
> @bldewolf <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_bldewolf&d=DwMCaQ&c=kbmfwr1Yojg42sGEpaQh5ofMHBeTl9EI2eaqQZhHbOU&r=C6Pt5AGtImanmAdcooarL-JZO8M5dSFPfs3VweYXYkE&m=76YnQZD8frvTIUgT6PNxt8TaUNadJY6fmGS7aTL1QLQ&s=K5PDR-n8jXmQRFKVmEAliuL8FaHkWhfQZ1IbUrARH1c&e=>, @mgage <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mgage&d=DwMCaQ&c=kbmfwr1Yojg42sGEpaQh5ofMHBeTl9EI2eaqQZhHbOU&r=C6Pt5AGtImanmAdcooarL-JZO8M5dSFPfs3VweYXYkE&m=76YnQZD8frvTIUgT6PNxt8TaUNadJY6fmGS7aTL1QLQ&s=w8dAGuYTRHdxrJJG2w9snfMLdrKXlRfCx5R4jq-kMmE&e=>, @dpvc <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_dpvc&d=DwMCaQ&c=kbmfwr1Yojg42sGEpaQh5ofMHBeTl9EI2eaqQZhHbOU&r=C6Pt5AGtImanmAdcooarL-JZO8M5dSFPfs3VweYXYkE&m=76YnQZD8frvTIUgT6PNxt8TaUNadJY6fmGS7aTL1QLQ&s=K3DD2gyslZB10Nvdwde8gA0dLV4tvcR2mOOXyHrbays&e=> I just read through the thread for #289 <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_openwebwork_pg_issues_289&d=DwMCaQ&c=kbmfwr1Yojg42sGEpaQh5ofMHBeTl9EI2eaqQZhHbOU&r=C6Pt5AGtImanmAdcooarL-JZO8M5dSFPfs3VweYXYkE&m=76YnQZD8frvTIUgT6PNxt8TaUNadJY6fmGS7aTL1QLQ&s=4f62e69UhaYtH5thSy092bIGWCWBp32z0u2E9AzN7Qc&e=>, do any of you a list of problems that generate the warnings, so I can test if these fix the warnings?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_openwebwork_pg_pull_290-23issuecomment-2D310198155&d=DwMCaQ&c=kbmfwr1Yojg42sGEpaQh5ofMHBeTl9EI2eaqQZhHbOU&r=C6Pt5AGtImanmAdcooarL-JZO8M5dSFPfs3VweYXYkE&m=76YnQZD8frvTIUgT6PNxt8TaUNadJY6fmGS7aTL1QLQ&s=GXj36frrYIVrgx8Qs3wxtxOCfQRV_ngszCHkWbig5CA&e=>, or mute the thread <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABJdfBAUStCaFyWv-2Dso680OUvOCnZf3Jks5sGX9dgaJpZM4LEzTT&d=DwMCaQ&c=kbmfwr1Yojg42sGEpaQh5ofMHBeTl9EI2eaqQZhHbOU&r=C6Pt5AGtImanmAdcooarL-JZO8M5dSFPfs3VweYXYkE&m=76YnQZD8frvTIUgT6PNxt8TaUNadJY6fmGS7aTL1QLQ&s=m9sCTo4NL83nucov0o7w757XEuoMlrczryocQ58N4Qs&e=>.
> 



Begin forwarded message:

**Comment by mgage** [comment]
A quicker answer to your question.  Grep the library for problems that include
niceTables.pl,  parserPopUp.pl, parserRadioButtons.pl and contextLimitedFactor.pl
Check a handful of those problems.

**Comment by pstaabp** [comment]
I've tested this now with a few problems with each of the macros and definitely the warnings are reduced.  From some of the problems, I'm still getting warnings, but it seems from the problem code itself, not the macros.   Cleaning up all of those would be quite a task... 

Also, looking at the code changes, they seem relatively innocuous.

**Comment by mgage** [comment]
Sounds like we should pull it.  As you say the code changes are straightforward.  Solving the other problems would not be quick so we'll kick that can down the road some more.

**Pull request merged**